src/utils/chunks/types.rs

```diff
@@ -17,7 +18,7 @@ pub type MissingObjectsInfo<'m, T> = (Vec<&'m Chunked<T>>, Vec<Chunk<'m>>);
 /// A trait for objects that can be assembled via the `assemble_difs` endpoint.
 pub trait Assemblable {
     /// Returns the name of the object.
-    fn name(&self) -> &str;
+    fn name(&self) -> Cow<str>;
```

Comment:
Why does this change? Is it ever useful to return an owned name from this trait?

Reply:
Answer: This is because in https://github.com/getsentry/sentry-cli/pull/2296 the name is constructed on the fly..